### PR TITLE
test

### PR DESCRIPTION
**Pull request by MateuszPapiez**

### CODE REVIEW

**Review by Peterses — APPROVED** [review]

File: Countries_application/Countries_application/ContentView.swift [hunk 1]
```diff
@@ -11,14 +11,65 @@ struct ContentView: View {
...
-            VStack {
-                Text("TODO:- country list")
-                
+            ZStack {
+                Color("Background")
+                    .ignoresSafeArea(.all)
                 
+                VStack {
+                    
+                    List{
```
Comment (Peterses) [inline Countries_application/Countries_application/ContentView.swift]: `List {`

File: Countries_application/Countries_application/ContentView.swift [hunk 2]
```diff
@@ -11,14 +11,65 @@ struct ContentView: View {
     var body: some View {
         
         NavigationView {
-            
-            VStack {
-                Text("TODO:- country list")
-                
+            ZStack {
+                Color("Background")
```
Comment (Peterses) [inline Countries_application/Countries_application/ContentView.swift]: Please create new file and store colors defined by you in `enum` or `struct`.

File: Countries_application/Countries_application/ContentView.swift [hunk 3]
```diff
@@ -11,14 +11,65 @@ struct ContentView: View {
...
+                                    .font(.system(size: 32))
+                                VStack(alignment: .leading, spacing: 1)
+                                {
+                                    Text("Polska")
+                                        .font(.system(size: 18))
+                                    Text("Warszawa")
+                                        .font(.system(size: 14))
+                                        .foregroundColor(.gray)
+                                    
+                                }
```
Comment (Peterses) [inline Countries_application/Countries_application/ContentView.swift]: This is ok, but you are coping same code for different country. Please create new view, for example `CountryElement` and pass country name, capital and flag as a parameters of init.

File: Countries_application/Countries_application/ContentView.swift [hunk 4]
```diff
@@ -11,14 +11,65 @@ struct ContentView: View {
...
+                                //padd
+                            }
+                        }
+                        HStack{
```
Comment (Peterses) [inline Countries_application/Countries_application/ContentView.swift]: `HStack {`

File: Countries_application/Countries_application/ContentView.swift [hunk 5]
```diff
@@ -11,14 +11,65 @@ struct ContentView: View {
...
+                                Text("Niemcy")
+                                    .font(.system(size: 18))
+                                Text("Berlin")
+                                    .font(.system(size: 14))
+                                    .foregroundColor(.gray)
+                            }
+                            
+                        }
+
+                        HStack{
```
Comment (Peterses) [inline Countries_application/Countries_application/ContentView.swift]: `HStack {`

File: Countries_application/Countries_application/CountryList.swift [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+//  Countries_application
+//
+//  Created by Mateusz Papież on 09/12/2022.
+//
+
+import SwiftUI
+
+struct Country: Identifiable {
+    let id = UUID()
+    let flag_image: String
```
Comment (Peterses) [inline Countries_application/Countries_application/CountryList.swift]: I know that's the preference but pls use in this project camel case while naming everything.

[https://www.theserverside.com/answer/Camel-case-vs-snake-case-Whats-the-difference?amp=1](https://www.theserverside.com/answer/Camel-case-vs-snake-case-Whats-the-difference?amp=1)

File: Countries_application/Countries_application/TestView.swift [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+//  Countries_application
+//
+//  Created by Mateusz Papież on 09/12/2022.
+//
+
+import SwiftUI
+
+struct TestView: View {
+    var body: some View {
+        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
```
Comment (Peterses) [inline Countries_application/Countries_application/TestView.swift]: What is it?

File: Countries_application/Countries_application/TestView.swift [hunk 8]
```diff
@@ -0,0 +1,20 @@
+//
+//  TestView.swift
+//  Countries_application
+//
+//  Created by Mateusz Papież on 09/12/2022.
+//
+
```
Comment (Peterses) [inline Countries_application/Countries_application/TestView.swift]: If you don't need that file pls remove it.

File: Countries_application/Countries_application/CountryList.swift [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+    let country_name: String
+    let capital: String
+}
+
+
+let country_lists: [Country] = [
+    Country(
+        //id: " ",
```
Comment (Peterses) [inline Countries_application/Countries_application/CountryList.swift]: Don't leave commented code like that.

File: Countries_application/Countries_application/CountryList.swift [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+        flag_image: "🇵🇱",
+        country_name: "Poland",
+        capital: "Warszawa"),
+    Country(
+        //id: " ",
```
Comment (Peterses) [inline Countries_application/Countries_application/CountryList.swift]: Don't leave commented code like that.

File: Countries_application/Countries_application/ContentView.swift [hunk 11]
```diff
@@ -11,14 +11,65 @@ struct ContentView: View {
...
+                                        .font(.system(size: 14))
+                                        .foregroundColor(.gray)
+                                    
+                                }
+                                //padd
+                            }
+                        }
+                        HStack{
+                            Text("🇩🇪")
+                                .font(.system(size: 32))
```
Comment (Peterses) [inline Countries_application/Countries_application/ContentView.swift]: Maybe you don't need to specify the font size directly? See font sizes already defined in SwiftUI. 

[https://developer.apple.com/design/human-interface-guidelines/foundations/typography/](https://developer.apple.com/design/human-interface-guidelines/foundations/typography/)

[https://developer.apple.com/documentation/swiftui/font](https://developer.apple.com/documentation/swiftui/font)

You can use it like that:
For example:
`.font(.headline)`

**Review by Peterses — DISMISSED** [review]

**Review by Peterses — CHANGES_REQUESTED** [review]
Pls fix comments

**Review by Peterses — APPROVED** [review]

**Comment by MateuszPapiez** [comment]
fix  error

**Pull request merged**